.gitlab-ci.yml

```diff
@@ -68,6 +68,13 @@ before_script:
   only:
     - triggers
 
+.run_when_testkitchen_triggered: &run_when_testkitchen_triggered
```

Comment:
can you add a comment similar to the ones on the other triggers?

In particular, `web` is only listed on this list, it'd be useful to explain what it's here for